### ISSUE TITLE
formatting TS code with use strict directives results in different (and incorrect!) code than direct use of prettier

### ISSUE THREAD

**Issue by bgoscinski** [esbenp/prettier-vscode#139]
### Repro code:
```ts
'use strict'

function test(a : string) {
  'use strict'
}
```

### prettier (expected) result:
```ts
"use strict";

function test(a: string) {
  "use strict";
}
```

https://prettier.github.io/prettier/#%7B%22content%22%3A%22'use%20strict'%5Cn%5Cnfunction%20test(a%20%3A%20string)%20%7B%5Cn%20%20'use%20strict'%5Cn%7D%5Cn%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22typescript%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%2C%22ast%22%3Afalse%7D%7D

### prettier-vscode result:
![prettier-vscode-bug](https://user-images.githubusercontent.com/5730169/27582257-93af6d58-5b30-11e7-843d-2cf94af95204.gif)

### info
It happens with any directives (I noticed it while using ngInject).
* vscode version: 1.13.1
* prettier-vscode version: 0.19.1
* OS: Windows 10 Pro
* locally installed TS version: 2.3.4
* prettier is not installed (only the one bundled with prettier-vscode)

**Comment by CiGit** [comment]
This is a known issue with prettier.
Fixed in prettier/prettier#2107 but not yet released

**Comment by bgoscinski** [comment]
Thanks for quick response :)
